### ISSUE TITLE
Updated revision didn't trigger a new build

### ISSUE THREAD

**Issue by joker-eph** [google/llvm-premerge-checks#96]
This update https://reviews.llvm.org/D72439#1816447 didn't trigger a build apparently (and the build was broken when this landed).
Any reason?

**Comment by ChristianKuehnel** [comment]
This looks like a duplicate of #84 as even the usual arc checks were not triggered. 

@dfki-jugr Did you upload the patch to Phabricator via arc or via the web interface?

My suspicion: If patches are uploaded via the web interface, this does (for whatever reason) not trigger a build.

**Comment by dfki-jugr** [comment]
@ChristianKuehnel Yes, you are right. We have uploaded our patch via the provided web interface.

**Comment by ChristianKuehnel** [comment]
That's strange. I just created a revision via the web ui and then it triggered a build: https://reviews.llvm.org/D72692

**Comment by ChristianKuehnel** [comment]
updated the documentation on this, not sure what else we could do there...

**Comment by joker-eph** [comment]
Thanks, that is quite unfortunate...

> not sure what else we could do there...

Can we send a patch to Phabricator? Is there another way to get notifications than Harbormaster?

**Comment by ChristianKuehnel** [comment]
I'm not sure if it makes sense to invest much time on Phabricator. I guess
we'll be moving to GitHub pull requests...

And we don't have anyone updating Phabricator right now.

I'm not aware of any other way of implementing triggers except Herald. But
I'm not an expert on Phabricator. If there is anything please let me know.

**Comment by joker-eph** [comment]
I am not confident about a move to GH pull-requests anytime soon, there was a strong opposition when I tried this topic last time...
There are valid concerns that I couldn't find an alternative to (Herald to begin with).

**Comment by ChristianKuehnel** [comment]
So here are some numbers:

- Percentage of Revisions coverd by pre-merge tests is increasing 😄 
- Number of builds where folder or person has not opted in is decreasing 😄 
- We have quite a stable number of Revisions that do not trigger any builds 😭 

<img width="1030" alt="Screenshot 2020-02-24 at 17 59 40" src="https://user-images.githubusercontent.com/19961352/75173825-c6496680-572f-11ea-8f43-d1148e6756ef.png">
